src/Symfony/Component/Form/Tests/Command/DebugCommandTest.php

```diff
@@ -186,6 +190,92 @@ class:%s
             , $tester->getDisplay(true));
     }
 
+    /**
+     * @dataProvider provideCompletionSuggestions
+     */
+    public function testComplete(array $input, array $expectedSuggestions)
+    {
+        $formRegistry = new FormRegistry([], new ResolvedFormTypeFactory());
+        $command = new DebugCommand($formRegistry);
+        $application = new Application();
+        $application->add($command);
+        $tester = new CommandCompletionTester($application->get('debug:form'));
+        $this->assertSame($expectedSuggestions, $tester->complete($input));
+    }
+
+    public function provideCompletionSuggestions(): iterable
+    {
+        yield 'option --format' => [
+            ['--format', ''],
+            ['txt', 'json'],
+        ];
+
+        yield 'form_type' => [
+            [''],
+            $this->getCoreTypes(),
+        ];
+
+        yield 'option for short name' => [
+            ['Symfony\\Component\\Form\\Extension\\Core\Type\\ButtonType', ''],
```

Comment:
Is this double backslash missing?
```suggestion
            ['Symfony\\Component\\Form\\Extension\\Core\\Type\\ButtonType', ''],
```

Reply:
Good catch, thanks!